cmd/issuer-rest/startcmd/start.go

```diff
@@ -211,10 +219,6 @@ const (
 	vcsClaimDataURLFlagUsage = "VCS claim data url. Format: https://HostName:Port."
 	vcsClaimDataURLEnvKey    = "VCS_CLAIM_DATA_URL"
 
-	vcsDemoIssuerFlagName  = "vcs-demo-issuer"
```

Comment:
Reason for removing this? was it unused?

Reply:
That indeed should not be touched 👍